modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -1420,8 +1427,7 @@ def test_xs():
     }
     df = pd.DataFrame(data=d)
     df = df.set_index(["class", "animal", "locomotion"])
-    with pytest.warns(UserWarning):
-        df.xs("mammal")
+    df.xs("mammal")
```

Comment:
this [still defaults to pandas](https://github.com/modin-project/modin/blob/689faeef9e4be7b8308703a3a57db0049a109e4f/modin/pandas/dataframe.py#L2398-L2404)

Reply:
Restored.